WalletWasabi/Blockchain/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -207,7 +207,7 @@ private ProcessedResult ProcessNoLock(SmartTransaction tx)
 						// remove double spent coins recursively (if other coin spends it, remove that too and so on), will add later if they came to our keys
 						foreach (SmartCoin doubleSpentCoin in doubleSpentSpenders)
```

Comment:
Here it would be better to iterate through `doubleSpentSpenders.Distinct(x => x.TransactionId)`

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/11555/commits/43cca65242dfffa8eafa113bd58391b17779c90d